### PR TITLE
movie: Breach (2020) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Zombie on a spaceship. Unremarkable. Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 1]
```diff
@@ -379,3 +379,48 @@ parental love [Richard pointedly proclaimed his love to his son Ethan.]
...
+boyfriend and girlfriend [Noah stowed away on the spaceship in large part to be with his pregnant girlfriend Hayley.]
+romantic love [Noah and and Hayley were two 20-somethings in love.]
+speculative spaceship [The story is set aboard a massive interstellar ark that was transporting 300,000 colonists to a habitable planet called New Earth.]
+extraterrestrial being [It turned out that the zombies were acting under the agency of a grotesque alien creature.]
+zombie [Everyone on the interstellar ark started turning into flesh eating, mindless zombies.]
+human vs. zombie [It was up to Clay, Noah and a handful of other crew members to fend off hoards of zombies that were running amok aboard the interstellar ark.]
+commander and subordinate [The no-nonsense Commander Stanley was pointedly strict with the crew members under his charge, especially Noah. Clay and his friends made moonshine behind the Commander Stanley's back.]
+
+:: Minor Themes
+expecting parents [While not much explored in the story, Noah had knocked up the admiral's daughter, Hayley, and they were expecting with child as a result.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:416]: Typo "expecting"

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 2]
```diff
@@ -379,3 +379,48 @@ parental love [Richard pointedly proclaimed his love to his son Ethan.]
...
+roommate and roommate [Noah and Clay were bunking in the same quarters.]
+cryogenic technology [The interstellar ark's 300,000 or so passengers and most of the crew were placed into cryogenic stasis at the outset of the voyage to New Earth.]
+hologram [The crew members were communicating remotely my means of holographic projections.]
+human occupational activity [The stowaway Noah impersonated a janitor and was shown working on the spaceship in that capacity.]
+misanthropy [Teek viewed humanity as a parasite and confessed to having been the one that released the turn-everyone-into-zombies parasite aboard the interstellar ark.]
+self-sacrifice [Admiral Adams blew himself up, taking out a number of zombies in the process, in an effort to give the others a chance of making it to New Earth.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:422]: Sounds like a https://www.themeontology.org/theme.php?name=sacrifice%20for%20one%27s%20people

**Review by odinlake — APPROVED** [review]

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt]: Good call. Updated and pushed accordingly.

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt:416]: I was expecting to find a typo in there, but I don't see.

**Comment by odinlake** [comment]
"they were expecting with child"

is not correct as far as I can parse it - I assumed the word "expecting" 
was surplus to requirement.

**Comment by paul-sheridan** [comment]
> "they were expecting with child" is not correct as far as I can parse it - I assumed the word "expecting" was surplus to requirement.
> […](#)
> On 09/07/2021 15:08, Paul Sheridan wrote: they were expecting with child

Ah I see what you're on about. Updated, pushed, and about to be merged.

**Pull request merged**